controllers/helpers.go

```diff
@@ -299,45 +302,134 @@ func GetSubnetUUID(ctx context.Context, client *prismclientv3.Client, peUUID str
 }
 
 // GetImageUUID returns the UUID of the image with the given name
-func GetImageUUID(ctx context.Context, client *prismclientv3.Client, imageName, imageUUID *string) (string, error) {
-	var foundImageUUID string
-
-	if imageUUID == nil && imageName == nil {
+func GetImageUUID(
+	ctx context.Context,
+	client *prismclientv3.Client,
+	imageName,
+	imageUUID,
+	imageLookupFormat,
+	imageLookupBaseOS,
+	k8sVersion *string,
+) (string, error) {
+	if imageLookupFormat == nil {
+		imageLookupFormat = ptr.To("nkp-{{.BaseOS}}-.*?{{.K8sVersion}}-.*")
```

Comment:
we can skip using `nkp` as it is internal acronym. Our CLI/UI can setup format, if this default is not good enough. 
```suggestion
		imageLookupFormat = ptr.To("*.{{.BaseOS}}-.*?{{.K8sVersion}}-.*")
```

We can also set it up in the API schema as default.

Reply:
yeah, that's a good idea. this is totally the wrong place to do it :sweat_smile:

Reply:
```suggestion
		imageLookupFormat = ptr.To("capx-{{.BaseOS}}-.*?{{.K8sVersion}}-.*")
```

I think it might be safer to default to the `capx` prefix. WDYT?

Reply:
changed the default format to use this!